plugins/jobs/src/js/components/JobsOverviewTable.js

```diff
@@ -162,8 +192,8 @@ class JobsTable extends React.Component {
       );
     }
 
-    if (schedules && schedules.length !== 0) {
-      const schedule = schedules[0];
+    if (schedules && schedules.nodes.length !== 0) {
```

Comment:
What if we have schedules, but schedules do not have nodes?

Reply:
`schedules` _always_ has `nodes`, otherwise its not a `JobSchedule`.